business_requirement_deliverable_resource_template/tests/test_br_resource_template.py

```diff
@@ -24,7 +24,7 @@ def setUp(self):
             'factor': 1,
             'uom_type': 'reference',
             'rounding': 0.000001})
-        self.ProductS = self.env.ref('product.product_product_consultant')
+        self.ProductS = self.env.ref('product.service_order_01')
```

Comment:
@seb-elico The product wasn't exist in v10 product module. So we changed it by other service type of product.